io.github.cudatext.CudaText-Qt5.yml

```
@@ -0,0 +1,110 @@
+      - sed -i -e "s/\$\$\[QT_INSTALL_LIBS\]/\"\\$FLATPAK_DEST\/lib\"/g" Qt5Pas.pro
+      - qmake "QT += x11extras"
+      - make
+      - make install
+
+  - name: cudatext
+    sources:
+      - type: archive
+        url: https://github.com/bgrabitmap/bgrabitmap/archive/refs/tags/v11.3.1.tar.gz
+        sha256: '9236aab15b3833908dffa8cc3fe7ebd9bce52f2b2845ede816235fb3b058401b'
+        dest: src/bgrabitmap
+      - type: archive
+        url: https://github.com/Alexey-T/EncConv/archive/refs/tags/2021.01.01.tar.gz
+        sha256: '98655e03eec2865fc32d8c6bd7ae9d2c96eb3b3cd895169963e62dfbd41dba85'
+        dest: src/EncConv
+      - type: archive
+        url: https://github.com/Alexey-T/ATBinHex-Lazarus/archive/refs/tags/2021.07.02.tar.gz
+        sha256: '9d33819e4cbf6258f2a35ef4cc7536b2e91bb4b884cbf15a99c80332ef308c44'
+        dest: src/ATBinHex-Lazarus
+      - type: archive
+        url: https://github.com/Alexey-T/ATFlatControls/archive/refs/tags/2021.07.09.tar.gz
+        sha256: '61aaacf795ffde0478c7e04dd9878d08e79e7c96f4f4ece10108e65403a36aaa'
+        dest: src/ATFlatControls
+      - type: archive
+        url: https://github.com/Alexey-T/ATSynEdit/archive/refs/tags/2021.07.29.tar.gz
+        sha256: '3ac5b87567eba0e70978ed90364b55cc954700f6a4ffb6112e09b1af09cd5843'
+        dest: src/ATSynEdit
+      - type: archive
+        url: https://github.com/Alexey-T/ATSynEdit_Cmp/archive/refs/tags/2021.07.20.tar.gz
+        sha256: '46a28b26b7ade095a48a5df698e072e5cadf34c8f1aafc8bad012899447ec6ed'
+        dest: src/ATSynEdit_Cmp
+      - type: archive
+        url: https://github.com/Alexey-T/EControl/archive/refs/tags/2021.07.29.tar.gz
+        sha256: 'aba831dc4d3eae34b2446bfa57dd6158ed9b85dcfb4f770a236bc5d9ef3defcf'
+        dest: src/EControl
+      - type: archive
+        url: https://github.com/Alexey-T/ATSynEdit_Ex/archive/refs/tags/2021.07.29.tar.gz
+        sha256: '69f47d7b69936dbb9ee16d4fbae7d113f5d06980de40003ace203f5a0e0e7728'
+        dest: src/ATSynEdit_Ex
+      - type: archive
+        url: https://github.com/Alexey-T/Python-for-Lazarus/archive/refs/tags/2021.04.16.tar.gz
+        sha256: '2b972e4fd714b774f22495afa22209ce8ae4f1282b5fa306bbccf20425fb4b27'
+        dest: src/Python-for-Lazarus
+      - type: archive
+        url: https://github.com/Alexey-T/Emmet-Pascal/archive/refs/tags/2020.09.05.tar.gz
+        sha256: 'cba0d68fee260a5f7aff51933014fe53ea3eb71f6d8349ff660b8b2a04028eaf'
+        dest: src/Emmet-Pascal
+      - type: archive
+        url: https://github.com/Alexey-T/CudaText/archive/refs/tags/1.139.5.tar.gz
+        sha256: '0c9929bf5870347399d017e104153de0a9a5bfd7428a8631e72bcf4481e2ea48'
+        dest: src/CudaText
+      - type: file
+        path: io.github.cudatext.CudaText-Qt5.appdata.xml
+    buildsystem: simple
+    build-commands:
+      - lazbuild --ws=qt5 -q --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/bgrabitmap/bgrabitmap/bgrabitmappack.lpk"
```

Found the problem. It was looking for `ppaarch64` instead of `ppa64`. So I override the compiler. Note that it's only needed once as subsequent calls to `lazbuild` will reuse the settings.

```suggestion
    build-options:
      arch:
        x86_64:
          env:
            PPC: ppcx64
        aarch64:
          env:
            PPC: ppca64
    build-commands:
      - lazbuild --ws=qt5 -q --compiler=$PPC --pcp=. --lazarusdir="$LAZARUS_DIR" "./src/bgrabitmap/bgrabitmap/bgrabitmappack.lpk"
```

The build succeed with that patch on my RaspberryPi

(you'll need to remove the flathub.json as well)

---

Thanks very much for figuring that out!